ArchiSteamFarm/ArchiSteamFarm.csproj

```diff
@@ -29,7 +29,7 @@
 		<PackageReference Include="System.Linq.Async" />
 	</ItemGroup>
 
-	<ItemGroup Condition="'$(TargetFramework)' != 'net48'">
+	<ItemGroup Condition="'$(TargetFramework)' != 'net48' AND ('$(TargetGeneric)' == 'true' OR '$(TargetWindows)' == 'true')">
```

Comment:
For completeness the additional condition should also be added to `Directory.Packages.props`.

Reply:
Good catch, thanks.